ooniprobe/Model/Settings/Settings.m

```diff
@@ -14,6 +47,11 @@ - (id)init {
         self.disabled_events = @[@"status.queued", @"status.update.websites", @"failure.report_close"];
         self.log_level = [SettingsUtility getVerbosity];
         self.options = [Options new];
+        NSArray *paths = NSSearchPathForDirectoriesInDomains(NSDocumentDirectory,NSUserDomainMask, YES);
+        NSString *documentsDirectory = [paths objectAtIndex:0];
+        self.state_dir = [documentsDirectory stringByAppendingPathComponent:[NSString stringWithFormat:@"state"]];
+        self.assets_dir = [documentsDirectory stringByAppendingPathComponent:[NSString stringWithFormat:@"assets"]];
```

Comment:
Can you put assets into a directory that can be easily cleaned by the user? The state directory really needs to be preserved, while the assets directory can be wiped if needed (we'll just download again if the user wipes it).

Reply:
We discuss it in https://github.com/ooni/probe-engine/issues/671
Short answer: there is not a directory that can be cleaned by the user, the iOS can clean some files in some circumstances but you as an app are responsible for cleanup.

Reply:
Yeah, I see what you mean. So let's put them in documentsDirectory and also update Android?